store/src/test/java/com/dropbox/android/external/store4/impl/FlowStoreTest.kt

```diff
@@ -444,6 +448,105 @@ class FlowStoreTest {
             )
     }
 
+    data class StringWrapper(val wrapped: String)
```

Comment:
why do we need this ?

Reply:
sorry leftover from some debugging. removing